perfkitbenchmarker/linux_benchmarks/cloudharmony_network_benchmark.py

```diff
@@ -306,6 +364,15 @@ def _Run(benchmark_spec, test):
     http_url = f'http://{benchmark_spec.bucket}.storage.googleapis.com/probe'
     endpoints = f'--test_endpoint={http_url}'
     _AddStorageMetadata(client, metadata)
+  elif FLAGS.ch_network_test_service_type == DNS:
+    vms = vm_groups['server']
+    # use GCP zonal internal DNS, but maybe should add domain to vm's data attributes?
+    endpoints = ''
+    if FLAGS.cloud == 'GCP':
+      endpoints = ' '.join([f'--test_endpoint={vm.name}.{vm.zone}.c.{vm.project}.internal' for vm in vms])
+    else:
+      raise NotImplementedError('DNS not implemented for this cloud type')
```

Comment:
is this a repeat of the check in checkprerequisites?

Reply:
Yes, I've removed the extra check